File: 비주얼 베이직/2019-1-간단한프로그램2-김유빈-pp71/2019-1-간단한프로그램2-김유빈-pp71/Form1.vb
Public Class Form1
    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click

        If (TextBox1.Text = "김유빈" And TextBox2.Text = "1122") Then
            Label3.Text = TextBox1.Text + " 님 반갑습니다"
        Else
            Label3.Text = "아이디와 비밀번호를 확인해주세요"

        End If
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        Label3.Text = "다시 입력해주세요."
        TextBox1.Text = ""
        TextBox2.Text = ""
        TextBox1.Focus()
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        End
    End Sub
End Class

File: C#/2019-2-C샵-3주차-구구단-김유빈/2019-2-C샵-3주차-구구단-김유빈/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace _2019_2_C샵_3주차_구구단_김유빈
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void button1_Click(object sender, EventArgs e)
        {
            int i, Dan;

            Dan = int.Parse(textBox1.Text);
            textBox2.Text = "";
            for (i = 1; i <= 9; i++)
            {
                textBox2.Text = textBox2.Text + Dan + " X " + i + " = " + Dan * i + Environment.NewLine;
            }
        }

        private void button2_Click(object sender, EventArgs e)
        {
            int i = 1, Dan;

            Dan = int.Parse(textBox4.Text);
            textBox3.Text = "";
            while (i <= 9)
            {
                textBox3.Text = textBox3.Text + Dan + " X " + i + " = " + Dan * i + Environment.NewLine;
                i++;
            }
        }

        private void button3_Click(object sender, EventArgs e)
        {
            int i = 1, Dan;

            Dan = int.Parse(textBox6.Text);
            textBox5.Text = "";
            do
            {
                textBox5.Text = textBox5.Text + Dan + " X " + i + " = " + Dan * i + Environment.NewLine;
                i++;
            }while (i <= 9) ;
        }

        private void button4_Click(object sender, EventArgs e)
        {
            int i = 1, Dan;

            Dan = int.Parse(textBox8.Text);
            textBox7.Text = "";
            while (true) //무한반복
            {
                textBox7.Text = textBox7.Text + Dan + " X " + i + " = " + Dan * i + Environment.NewLine;
                i++;
                if (i > 9)
                    break;
            }
        }
    }
}
